### PR TITLE
Add a retry loop in the getContents test to work around a known bug r…

### PR DESCRIPTION
**Pull request by DenverM80**
…egarding job status.

### CODE REVIEW

File: ds3-sdk-integration/src/test/java/com/spectralogic/ds3client/integration/Smoke_Test.java [hunk 1]
```diff
@@ -253,9 +254,21 @@ public SeekableByteChannel buildChannel(final String key) throws IOException {
...
+            JobStatus getJobResponseStatus = JobStatus.IN_PROGRESS;
+            for (int retry = 0; retry < 3; retry++) {
+                final GetJobResponse jobResponse = client.getJob(new GetJobRequest(jobId));
+                getJobResponseStatus = jobResponse.getMasterObjectList().getStatus();
 
+                if (getJobResponseStatus == JobStatus.COMPLETED) break;
+
+                try {
+                    TimeUnit.MILLISECONDS.sleep(500);
+                } catch (InterruptedException e) {
```
Comment (RachelTucker) [inline ds3-sdk-integration/src/test/java/com/spectralogic/ds3client/integration/Smoke_Test.java]: missing final

Comment (DenverM80) [inline ds3-sdk-integration/src/test/java/com/spectralogic/ds3client/integration/Smoke_Test.java]: odd, I thought I'd enabled that as a warning in IntelliJ.  Good catch, thanks!

Comment (DenverM80) [inline ds3-sdk-integration/src/test/java/com/spectralogic/ds3client/integration/Smoke_Test.java]: updated

**Comment by RachelTucker** [comment]
Looks good

**Comment by DenverM80** [comment]
http://jira.eng.sldomain.com/browse/JSDK-94

**Comment by rpmoore** [comment]
The pull request is now out of date.

**Comment by DenverM80** [comment]
updated

**Pull request merged**